src/services/KeyManager/index.js

```diff
@@ -1,55 +1,69 @@
 import {Keyring} from '@polkadot/api';
+import {KeyringPair} from '@polkadot/keyring/types';
 import {
   mnemonicGenerate,
   decodeAddress,
   blake2AsHex
 } from '@polkadot/util-crypto';
 import {stringToU8a, u8aToHex} from '@polkadot/util';
+import {BehaviorSubject} from 'rxjs';
 
 class KeyManager {
+  /**
+   * @type {BehaviorSubject<KeyringPair>}
+   * @private
+   */
+  _pair = new BehaviorSubject(null);
+  pair$ = this._pair.asObservable();
```

Comment:
I'm sure you've got a solid reason but what's the thought behind switching this to an Observable?

Reply:
so that when the account is loaded, the `useAccount` hook will react to it since it subscribed to the observable in the `useEffect` hook